app/scripts/modules/core/src/delivery/create/TemplateSelector.tsx

```diff
@@ -25,7 +23,7 @@ export class TemplateSelector extends React.Component<IProps, IState> {
           <Select
             options={this.createOptionsFromTemplates()}
             clearable={true}
-            value={selected ? {label: selected.metadata.name, id: selected.id} : null}
+            value={selected ? {label: selected.metadata.name, value: selected.id} : null}
```

Comment:
@danielpeach PTAL
I think this was supposed to be ` {label: selected.metadata.name, value: selected.id}`.  My IDE was not complaining, but `tsc` was.